perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -52,101 +52,6 @@ C<0001-01-01T00:00:00> represents an undefined date in Whitespace.
 
 use constant WHITESPACE_UNDEF_DATE => '0001-01-01T00:00:00';
 
-sub waste_fetch_events {
```

Comment:
This method is called by the `bin/fixmystreet.com/fetch-waste` script, which is used when there's a need to manually pull events from Whitespace, in case a push is missed or similar.

Would be good if we could keep this method working so we've got the option to pull updates, similar to how the Echo ones work.

Reply:
I've broken everything down into separate commits rather than just fixups to try and keep this clear as it was a lot of changes moving back into the files. Let me know if it's not clear.

I have added a fixup at the end as I think that I had overlooked the comment id is probably now actually supplied in the data if I've understood the data correctly.

Reply:
The worksheet reference is our report ID, so it's not a unique update ID. We could concatenate the worksheet ID and the timestamp?

Reply:
Or just use 'waste' as that is what Echo use for the update_id, to match.